gen/nvml/gpm.go

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package nvml
+
+// nvml.GpmMetricsGet()
```

Comment:
Is this comment at the right place?

Reply:
Yeah. I wanted it at the top of the full set of functions related to GpmMetricsGet(). I could move the actual GpmMetricsGet() to the top though if you think that's clearer.

Reply:
I would think it's a little clearer in terms of docstring-like usage. It's not a blocker though.